internal/database/election.go

```diff
@@ -12,12 +12,11 @@ import (
 type Election struct {
```

Comment:
Consider adding an explicit interface guard near the definition to make it obvious that this should implement `Pollable`, eg:

```go
var _ Pollable = (*Election)(nil)
```